aim/web/ui/src/pages/RunDetail/RunSelectPopoverContent.tsx

```diff
@@ -130,18 +130,46 @@ function RunSelectPopoverContent({
                       to={pathname.replace(runHash, run.run_id)}
                       onClick={onRunsSelectToggle}
                     >
-                      <Text
-                        size={14}
-                        tint={runInfo?.name === run.name ? 100 : 80}
-                        weight={runInfo?.name === run.name ? 600 : 500}
+                      <Tooltip title={run.name} placement='right'>
+                        <div
+                          className={
+                            'RunSelectPopoverWrapper__selectPopoverContent__contentContainer__runsListContainer__runsList__runBox__runName'
+                          }
+                        >
+                          <Text
+                            size={14}
+                            tint={runInfo?.name === run.name ? 100 : 80}
+                            weight={500}
+                          >
+                            {run.name}
```

Comment:
Why showing run name in tooltip, maybe tooltip should include some other information, since the run name is already shown on the navigation item?

Reply:
If the run name is too long, a user would be able to see it in the tooltip.